packer-assets/ubuntu-trusty-ci-amethyst-packages.txt

```diff
@@ -117,11 +116,9 @@ klibc-utils
 kmod
 krb5-locales
 language-pack-en
-language-selector-common
```

Comment:
All of the packages that have been removed in this PR were left over from a previous PR that sought to establish a baseline of packages with high overlap between packer images built on GCE and docker.  It appears I merged the previous PR prematurely 🙈.  These packages have been removed because they are purged prior to running the serverspec suite(s).